Slim/Utils/Prefs.pm

```diff
@@ -191,6 +191,8 @@ sub init {
 		'releaseTypesToIgnore'  => [],
 		'ignoreReleaseTypes'    => 0,
 		'groupArtistAlbumsByReleaseType' => 0,
+		'showComposerReleasesbyAlbum' => 2,
+		'showComposerReleasesbyAlbumGenres' => "Classical",
```

Comment:
Are genre tag values always localised? even if they are, I can imagine people having music files which do not match their language preference in LMS. 

The idea of this was to flag the enhancement to people by changing the grouping behaviour for classical releases by default, rather than leave the feature undiscovered for people who don't read the release notes.

Maybe we could initialise to "classical,classique,..." ie a few languages. I'll find out what genre tag values in different languages could be for classical releases.

Reply:
Yes, I believe having a few localized versions (EN, DE, NL, FR are the most important languages FWICT - with a huge gap after EN & DE) is probably the easiest way to go. After all that would only be like four words to start with.